From 72f3cf3b2e4fd62aa86fcc20df93b07247302a3e Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Wed, 3 Jun 2026 13:40:01 +0200
Subject: [PATCH] uboot-tools: fix compilation error due to missing setuptools

Added setuptools/host as dependency to fix compilation.
Fixing:
Traceback (most recent call last):
 File "/.../build_dir/target-arm_cortex-a9+vfpv3-d16_musl_eabi/u-boot-2025.10/scripts/dtc/pylibfdt/setup.py", line 23, in <module>
    from setuptools import setup, Extension
ModuleNotFoundError: No module named 'setuptools'
---
 package/boot/uboot-tools/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/boot/uboot-tools/Makefile b/package/boot/uboot-tools/Makefile
index dfacb6f330..283fc82ece 100644
--- a/package/boot/uboot-tools/Makefile
+++ b/package/boot/uboot-tools/Makefile
@@ -14,7 +14,7 @@ PKG_HASH:=b4f032848e56cc8f213ad59f9132c084dbbb632bc29176d024e58220e0efdf4a
 PKG_SOURCE_SUBDIR:=$(PKG_DISTNAME)-$(PKG_VERSION)
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_DISTNAME)-$(PKG_VERSION)
 
-PKG_BUILD_DEPENDS:=fstools python3/host
+PKG_BUILD_DEPENDS:=fstools python3/host python-setuptools/host
 
 PKG_LICENSE:=GPL-2.0 GPL-2.0+
 PKG_LICENSE_FILES:=Licenses/README
-- 
2.54.0

